storage/rocksdb/ha_rocksdb.cc

```diff
@@ -412,6 +412,10 @@ static constexpr ulong RDB_MAX_LOCK_WAIT_SECONDS = 1024 * 1024 * 1024;
 static constexpr ulong RDB_MAX_ROW_LOCKS = 1024 * 1024 * 1024;
 static constexpr ulong RDB_DEFAULT_ROW_LOCKS = 1024 * 1024;
 static constexpr ulong RDB_DEFAULT_BULK_LOAD_SIZE = 1000;
+static constexpr int RDB_DEFAULT_MAX_BACKGROUND_JOBS = 2;
```

Comment:
Strictly speaking each "constexpr" has to be "const constexpr" here but meh

Reply:
true, I'll JIRA this to review these as well as TokuDB later and fix them up.
https://jira.percona.com/browse/PS-4964